data.sql

```diff
@@ -1,14 +1,35 @@
 /* Populate database with sample data. */
 
 
-insert into animals (id, name, date_of_birth, escape_attempts ,  neutered, weight_kg )
-values (1, 'Agumon', '2020-02-03', 0, true, 10.23)
+insert into animals (name, date_of_birth, escape_attempts ,  neutered, weight_kg )
+values ('Agumon', '2020-02-03', 0, true, 10.23)
 
-insert into animals (id, name, date_of_birth, escape_attempts ,  neutered, weight_kg )
-values (2,  'Gabumon', '2018-11-15', 2, true, 8) 
+insert into animals (name, date_of_birth, escape_attempts ,  neutered, weight_kg )
+values ('Gabumon', '2018-11-15', 2, true, 8) 
 
-insert into animals (id, name, date_of_birth, escape_attempts ,  neutered, weight_kg )
-values (3,  'Pikachu', '2021-01-07', 1, false, 15.0) 
+insert into animals (name, date_of_birth, escape_attempts ,  neutered, weight_kg )
+values ('Pikachu', '2021-01-07', 1, false, 15.0) 
 
-insert into animals (id, name, date_of_birth, escape_attempts ,  neutered, weight_kg )
-values (4,  'Devimon', '2017-06-12', 5, true, 11)
\ No newline at end of file
+insert into animals (name, date_of_birth, escape_attempts ,  neutered, weight_kg )
+values ('Devimon', '2017-06-12', 5, true, 11)
+
+insert into animals (name, date_of_birth, escape_attempts ,  neutered, weight_kg)
+values ('Charmander', '2020-02-08', 0, False, 11);
```

Comment:
- [ ] The weight of "Charmander" is -11kg not 11kg. 

![image](https://user-images.githubusercontent.com/75126481/220427863-288326ba-c18d-45fa-9004-679945341711.png)